spring-xd-related-work.tex

```diff
@@ -49,15 +49,65 @@ \subsection{Spring XD and Storm}
 \item Building upon the functional stream processing model, users have the option
 to choose from Reactor\cite{reactor}, Spark Streaming or RxJava APIs to build
 complex data centric applications.
+\item As an alternative to linear streaming data pipelines, Spring XD provides 
+support for directed ayclic graphs (DAG)\cite{dag} through named channels to 
```

Comment:
I think that a critical difference is that Storm's deployment unit is the Topology/DAG, whereas in our case it is the linear stream that is hinted at the end of the paragraph, but maybe it needs to be better outlined.